Packs/Sixgill/Integrations/Sixgill/Sixgill.yml

```diff
@@ -0,0 +1,81 @@
+category: Data Enrichment & Threat Intelligence
+fromversion: 5.0.0
+commonfields:
+  id: Sixgill
+  version: -1
+configuration:
+- display: Sixgill API client ID
+  name: client_id
+  required: true
+  type: 0
+- display: Sixgill API client secret
+  name: client_secret
+  required: true
+  type: 4
+- display: Max number of incidents that can be fetched
+  name: maxIncidents
+  required: false
+  type: 0
```

Comment:
add here a initial value
currently it is empty and not explanatory 
![image](https://user-images.githubusercontent.com/7270217/73282673-aa1edc00-41fa-11ea-8567-087585c66aab.png)